notebooks/notebook_template.ipynb

```diff
@@ -31,6 +31,8 @@
       "source": [
```

Comment:
<p>Enabling APIs are specific to each notebook. Is it now moved to the separate set up guide? I think it would be better to add those commands here. For example:</p><p><code>gcloud services enable dataflow.googleapis.com</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I will revert lines 1 to 4 for now as they are important enough.</p>